db/models/user.js

```diff
@@ -8,15 +8,26 @@ const User = db.define('users', {
   username: Sequelize.STRING,
   first_name: Sequelize.STRING,
   last_name: Sequelize.STRING,
+  phone_number: Sequelize.STRING,
```

Comment:
Again, I'm also sort of personally iffy as to whether or not this is necessary...and for all of admin stuff, we can put that somewhere else?